packages/contracts-bedrock/src/L1/DelayedVetoable.sol

```diff
@@ -42,17 +43,11 @@ contract DelayedVetoable is ISemver {
     /// @param data The data forwarded to the target.
     event Vetoed(bytes32 indexed callHash, bytes data);
 
-    /// @notice The address that all calls are forwarded to after the delay.
-    address internal immutable TARGET;
+    /// @notice The target for calls from this contract.
+    address internal TARGET;
 
-    /// @notice The address that can veto a call.
-    address internal immutable VETOER;
-
-    /// @notice The address that can initiate a call.
-    address internal immutable INITIATOR;
-
-    /// @notice The delay which will be set after the initial system deployment is completed.
-    uint256 internal immutable OPERATING_DELAY;
+    /// @notice The superchain config contract.
+    SuperchainConfig internal _superchainConfig;
```

Comment:
No, good catch. It can be made immutable. .
Same goes for the `TARGET` var above. It looks like I put it into storage while moving the other immutables (`VETOER`, `INITIATATOR`, etc.) to the SuperchainConfig.